JS/inject.js

```diff
@@ -1,64 +1,73 @@
 class MessageData {
     constructor(fullContent = "a,b,c") { //signal, service, content
         var fullContent = fullContent.split(",");
-        this.signal = fullContent[0];
+        this.signal = fullContent[0]; //signal should be "studio721" to signify the message's purpose
         this.service = fullContent[1];
         this.content = fullContent[2];
     }
 }
 
+var lastSentVideo = "";
 var player,
     time_update_interval = 0;
+
+function showBottomBar(){
+    if(document.querySelectorAll(".fT3JUc").length > 0){
+        document.querySelector(".fT3JUc").classList.add("LCXT6");
+    }
+}
+
+showBottomBar();
+
+setInterval(function(){
+    showBottomBar()
+}, 1000);
+
 document.querySelectorAll(".l4V7wb")[0].addEventListener("mouseup", function () {
```

Comment:
Can we add this querySelector as a named constant so we know what we are looking for in case it changes?